### PR TITLE
Adding Defaults For Fog Settings

### LINKED ISSUE #283: Choose Better Defaults For Fog Settings
Currently, `fogStart` and `fogEnd` default to `0` with the new `LevelInfo` object (introduced in https://github.com/Interrupt/delverengine/pull/272).

grafik

This causes lighting issues.

grafik

We should find good default values and apply them.

### PR DESCRIPTION
**Pull request by PythooonUser**
Fixes #283 by introducing defaults for `fogStart` and `fogEnd`.

### CODE REVIEW

**Review by joshuaskelly — APPROVED** [review]
Looks good!

File: Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java [hunk 1]
```diff
@@ -23,11 +23,11 @@ public class LevelInfo extends Entity {
 
     /** Starting distance of fog. */
     @EditorProperty
-    public static float fogStart;
+    public static float fogStart = 1f;
```
[the remark this answers is not in the document]
Comment (PythooonUser) [reply Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java]: Used `0f`.

File: Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java [hunk 2]
```diff
@@ -23,11 +23,11 @@ public class LevelInfo extends Entity {
...
 
     /** Ending distance of fog. */
     @EditorProperty
-    public static float fogEnd;
+    public static float fogEnd = 34f;
```
[the remark this answers is not in the document]
Comment (PythooonUser) [reply Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java]: I put it at `15f`. Still too high?

[the remark this answers is not in the document]
Comment (PythooonUser) [reply Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java]: Closing.

**Comment by PythooonUser** [comment]
Thanks :tada:

**Pull request merged**